jena-arq/src/main/java/org/apache/jena/riot/system/RiotLib.java

```diff
@@ -18,20 +18,6 @@
 
 package org.apache.jena.riot.system;
 
-import static org.apache.jena.riot.RDFLanguages.NQUADS;
```

Comment:
Yes indeed, that would produce a lot of noise.
IDEs can import/export style rules, I feel like it would help with first PRs to have such rules delivered with the source code.

Reply:
The code is mixture of styles from many years of evolution. We don't have hard-and-fast rules; reformatting clutters up history and Jena has retain history from most of its life, even back to into SVN times (but not CVS). So generally, it's reflect the style of the file.